var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -12,7 +12,7 @@ class PyBglibpy(PythonPackage):
     git      = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
 
     version('develop', branch='master')
-    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545', preferred=True)
+    version('4.0.28', commit='9205075489845fc389b2f2f8e9637dfa14345ee6', preferred=True)
```

Comment:
Could we use tags, please?

Reply:
I'm not in charge of the bglibpy project, and it only has tags for major versions.  It uses versioneer, so I don't know how that works with tags.

Reply:
Maybe @wvangeit can comment

Reply:
See here https://github.com/BlueBrain/spack/pull/419

Reply:
Also, please don't remove old versions while they are still being referenced as dependencies. There's no harm in leaving a (maybe short-ish) list of versions.

Reply:
With python3, the `4.0.17` wouldn't work, so I don't think it's worth leaving around.

Reply:
I agree with @mgeplf